server/startup/migrations/v229.js

```diff
@@ -0,0 +1,36 @@
+import { Settings } from '../../../app/models/server';
+import { Migrations } from '../../../app/migrations';
+
+Migrations.add({
+	version: 229,
+	up() {
+		const oldNamesValidationSetting = Settings.findById(
```

Comment:
looking the lines below I can see they expect this to be a `findOne`, since it returns the record.. using `findById` will return a cursor, that needs to be `fetch()`.
```suggestion
		const oldNamesValidationSetting = Settings.findOneById(
```

Reply:
added the change and thanks for the tip!